typescript/test/test_application/karma.conf.js

```diff
@@ -0,0 +1,102 @@
+/**
```

Comment:
I'm not checking this file, it might needs to be changed when we check the browser use case but for now let's just commit it as is.

Reply:
Left a TODO comment to track the undone work.